packages/framework/src/types/schema.types.ts

```diff
@@ -3,8 +3,37 @@ import zod from 'zod';
 
 export type JsonSchema = JSONSchema;
 
+/**
+ * A JSONSchema or a ZodSchema.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** Not sure that this comment clarifies anything -- will we just add another "or ___" when we add a new schema type? Instead, perhaps we can explain what the purpose is of a `Schema`?‏

Reply:
Great callout, I added a description and converted into a list view for easier extensibility.